components/AppBar/src/MDCAppBar.m

```diff
@@ -151,7 +151,7 @@ - (MDCHeaderStackView *)headerStackView {
   // Underlying issue is you need view loaded before accessing. Below change will accomplish that
   // by calling for view.bounds initializing the stack view
```

Comment:
If we are pulling out self.view, should we also update the comment immediately above?